Dockerfile.processing_image

```diff
@@ -1,6 +1,6 @@
 ARG BASE_TAG=branch-main
 
-FROM ghcr.io/chanzuckerberg/corpora-upload-base:$BASE_TAG
+FROM ghcr.io/chanzuckerberg/corpora-upload-base:20211215
```

Comment:
We should probably add a comment here indicating why we're pinning to this magic version of the base image.